include/libtorrent/kademlia/dht_tracker.hpp

```diff
@@ -69,14 +71,18 @@ namespace libtorrent { namespace dht {
 			, counters& cnt
 			, dht_storage_interface& storage
 			, dht_state state);
-		virtual ~dht_tracker();
+
+		dht_tracker(dht_tracker const&) = delete;
```

Comment:
Is this a "noncopyable" thing? if that's the case, don't you need `operator=(dht_tracker const&) = delete` for consistency?